docs/app/pages/components/components-sections/tables/column-picker/column-picker.component.html

```diff
@@ -79,6 +89,42 @@
     </tr>
 </uxd-api-properties>
 
+<uxd-api-properties tableTitle="ColumnPickerGroupItem Interface">
+    <tr uxd-api-property name="group" type="string">
+        The name of the group that this column belongs to.
+    </tr>
+    <tr uxd-api-property name="name" type="string">
+        The name of the column.
+    </tr>
+</uxd-api-properties>
+
+<uxd-api-properties tableTitle="ColumnPickerGroupSetting Interface">
+    <tr uxd-api-property name="group" type="string">
+        The name of the group this setting object is related to.
+    </tr>
+    <tr uxd-api-property name="initiallyExpanded" type="boolean">
```

Comment:
What are the thoughts on this `groupSettings` object changing over time. Eg:

1. The user may want to programmatically expand a group after initial load, in which case we need to add some logic to ngOnChanges to perform this change when the input changes.
2. Could the user ever want to know when the expanded state changes? In which case an event that is emitted on change with support for two way binding?

Regardless of whether we implement this now or not, we probably should rename `initiallyExpanded` to just `expanded` as this would allow us to make these changes at some point in the future without having to rename this property.

Thoughts? @donna-marijne

Reply:
It's certainly a possible use case. I think if we were planning to ever make this collection more interactive then it would be better to drop "settings" from the name and associated type. In the future we could add a `groupsChange` output when the state changes.

Reply:
@ashley-hunter @donna-marijne I agree. I'll rename the property mentioned above.